rmgpy/molecule/pathfinder.py

```diff
@@ -480,3 +480,51 @@ def is_atom_able_to_lose_lone_pair(atom):
     return (((atom.is_nitrogen() or atom.is_sulfur()) and atom.lone_pairs in [1, 2, 3])
             or (atom.is_oxygen() and atom.lone_pairs in [2, 3])
             or atom.is_carbon() and atom.lone_pairs == 1)
+
+def find_adsorbate_delocalization_paths(atom1):
+    """
+    Find all bidentate adsorbates which have a bonding configuration X-C-C-X.
+    Examples:
+
+    - XCXC, XCHXCH, XCXCH, where X is the surface site
```

Comment:
What is the meaning of, e.g., `XCXC`, is the second X connected to the 1st or 2nd carbon? Or maybe it doesn't matter?

Reply:
The notation for (bidentate) adsorbates is a little tricky and there is no real consensus in the literature. I always place the adsorption site (the 'X') on the left side of the adatom, e.g. XC means that a carbon atom is bonded to the surface and XCXC means that the first and the second carbon atom have a bond to the surface. That said, CXCX or XCCX would be perfectly fine, too. I add some text to explain my notation.